app/modules/ads/tracking/jwplayer-tracker.js

```diff
@@ -63,6 +63,24 @@ export default class JWPlayerTracker {
     Object.keys(trackingEventsMap).forEach((playerEvent) => {
       player.on(playerEvent, (event) => {
         let errorCode;
+        const vastParams = event.tag ? vastParser.parse(event.tag) : null;
```

Comment:
As far as I understand we need to have those flags on `init`/`load` events. So we can't base on vast request beacuse it happens to late. Can't we use [playerState](https://github.com/Wikia/app/blob/dev/extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.ads.js#L105) for that?

Reply:
You right. Added some code, now status is properly set on `ready` event (`init` is too soon).